### ISSUE TITLE
Cannot set default value in date/time pickers.

### ISSUE THREAD

**Issue by cklog** [elementalui/elemental#69]
First and foremost, thank you for this beautiful resource.

By setting the Form Input type to "date" or "time", I was successful in rendering both these fields. I could not, however, set default values.

Moreover, there doesn't seem to be any documentation referring to these fields on the Elemental example page. I only discovered them by testing different field types. Trawling through past issues, it seems this feature was supported but is since deprecated.

Any leads on this front would be appreciated.

**Comment by NogsMPLS** [comment]
```<input type="date" />``` and ```<input type="time">``` are only really supported in Chrome & Opera, which is probably why you were able to see 'something' when you did that. (See: http://caniuse.com/#feat=input-datetime ).

If you try in pretty much any other browser though, those two input types will not work.

You are correct in that it seems like Elemental has deprecated/halted development on the date-picker, as there are some pieces of it sitting around in the code. I believe it is not documented because it is not ready.

So in short, the date picker and time picker you are seeing are native html 5 types that only chrome supports, and nothing to do with Elemental itself.

**Comment by jossmac** [comment]
Unrelated to Elemental UI
